### PR TITLE
ja: Make tutorials/hello-minikube.md follow v1.17 of the original text

### PR DESCRIPTION
**Pull request by ryokotmng**
Issue https://github.com/kubernetes/website/issues/21366

I referred [this doc](https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b) to fix.

### What I'm not pretty confident
I may have to change the 'Output is similar to:' wording to emphasize that users possibly see different output in some part?

### Appreciate
Since it's my very first PR for OSS, would be great if reviewers let me know any ideas and preferences where and how to fix or what to do if I missed something, even the smallest details. Thanks you vm for your review.

### CODE REVIEW

File: content/ja/docs/tutorials/hello-minikube.md [hunk 1]
```diff
@@ -109,19 +110,19 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
     kubectl config view
     ```
-  
+
     {{< note >}} `kubectl`コマンドの詳細な情報は[kubectl overview](/docs/user-guide/kubectl-overview/)を参照してください。{{< /note >}}
 
 ## Serviceの作成
 
-通常、PodはKubernetesクラスタ内部のIPアドレスからのみアクセスすることができます。`hello-node`コンテナをKubernetesの仮想ネットワークの外部からアクセスするためには、Kubernetesの[*Service*](/ja/docs/concepts/services-networking/service/)としてポッドを公開する必要があります。
+通常、PodはKubernetesクラスタ内部のIPアドレスからのみアクセスすることができます。`hello-node`コンテナをKubernetesの仮想ネットワークの外部からアクセスするためには、Kubernetesの[*Service*](/ja/docs/concepts/services-networking/service/)としてPodを公開する必要があります。
```
Comment (ryokotmng) [inline content/ja/docs/tutorials/hello-minikube.md:118]: changed 'ポッド' -> Pod

**Comment by oke-py** [comment]
@ryokotmng Thank you for contribution!
It seems that the PR title and referenced issue number are wrong.
Could you edit the PR title and description?

Additionally, please comment `/assign` in the related issue to assign it to you.

**Comment by ryokotmng** [comment]
Ahhhh, thank you for the advise. Will do quickly!

**Comment by ryokotmng** [comment]
I think now the title and description are correct, and changed the commit message of the first commit of this PR as well. And I added myself to the issue. Thanks in advance for your review:)

**Comment by oke-py** [comment]
@ryokotmng Thank you, but the description's 1st line is still wrong.
If you write only `#21366` instead of `https://github.com/kubernetes/website/issues/21366`, it will work.

**Comment by ryokotmng** [comment]
Apologies for my mistake🙇‍♂️ Now fixed!

**Comment by oke-py** [comment]
Good!

File: content/ja/docs/tutorials/hello-minikube.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ card:
 
 {{% capture overview %}}
 
-このチュートリアルでは、[Minikube](/docs/getting-started-guides/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
+このチュートリアルでは、[Minikube](/docs/setup/learning-environment/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:18]:
```suggestion
このチュートリアルでは、[Minikube](/ja/docs/setup/learning-environment/minikube)とKatacodaを使用して、Kubernetes上でシンプルなHello WorldのNode.jsアプリケーションを動かす方法を紹介します。Katacodaはブラウザで無償のKubernetes環境を提供します。
```
翻訳済みのためjaにリンクしておきます。

File: content/ja/docs/tutorials/hello-minikube.md [hunk 3]
```diff
@@ -79,19 +79,20 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
     kubectl get deployments
     ```
 
-    出力:
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:84]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L59

File: content/ja/docs/tutorials/hello-minikube.md [hunk 4]
```diff
@@ -79,19 +79,20 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
 3. Podを確認します:
 
     ```shell
     kubectl get pods
     ```
-    出力:
+
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:97]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L74

File: content/ja/docs/tutorials/hello-minikube.md [hunk 5]
```diff
@@ -130,7 +131,7 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
     kubectl get services
     ```
 
-    出力:
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:136]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L106

File: content/ja/docs/tutorials/hello-minikube.md [hunk 6]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
 
 4. Katacoda環境のみ：ターミナル画面上部の+ボタンをクリックして **Select port to view on Host 1** をクリックしてください。
 
-5. Katacoda環境のみ：`30369`(Service出力に表示されている`8080`の反対側のポートを参照)を入力し、クリックしてください。
+5. Katacoda環境のみ：`8080`の反対側のService出力に、5桁のポート番号が表示されます。このポート番号はランダムに生成されるため、ここで使用するポート番号と異なる場合があります。ポート番号テキストボックスに番号を入力し、ポートの表示をクリックしてください。前の例の場合は、「30369」と入力します。
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:153]:
```suggestion
5. Katacoda環境のみ：`8080`の反対側のService出力に、5桁のポート番号が表示されます。このポート番号はランダムに生成されるため、ここで使用するポート番号と異なる場合があります。ポート番号テキストボックスに番号を入力し、ポートの表示をクリックしてください。前の例の場合は、`30369`と入力します。
```

File: content/ja/docs/tutorials/hello-minikube.md [hunk 7]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
 1. サポートされているアドオンをリストアップします:
 
     ```shell
     minikube addons list
     ```
 
-    出力:
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:169]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L134

File: content/ja/docs/tutorials/hello-minikube.md [hunk 8]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
     nvidia-driver-installer: disabled
     nvidia-gpu-device-plugin: disabled
     registry: disabled
     registry-creds: disabled
     storage-provisioner: enabled
+    storage-provisioner-gluster: disabled
     ```
-   
+
 2. ここでは例として`heapster`のアドオンを有効化します:
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:189]:
```suggestion
2. ここでは例として`metrics-server`のアドオンを有効化します:
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L158-L160

File: content/ja/docs/tutorials/hello-minikube.md [hunk 9]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
 
     ```shell
     minikube addons enable heapster
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:192]:
```suggestion
    minikube addons enable metrics-server
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L163-L164

File: content/ja/docs/tutorials/hello-minikube.md [hunk 10]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
     ```
-  
-    出力:
+
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:197]:
```suggestion
    ```
```

File: content/ja/docs/tutorials/hello-minikube.md [hunk 11]
```diff
@@ -149,53 +150,55 @@ Kubernetesの[*Pod*](/ja/docs/concepts/workloads/pods/pod/) は、コンテナ
...
     heapster was successfully enabled
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:198]:
```suggestion
    metrics-server was successfully enabled
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L169-L175

File: content/ja/docs/tutorials/hello-minikube.md [hunk 12]
```diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:209]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L185-L186

File: content/ja/docs/tutorials/hello-minikube.md [hunk 13]
```diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
...
-    minikube addons disable heapster
+    minikube addons disable metrics-server
     ```
 
-    出力:
+    出力は下記のようになります:
 
     ```shell
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:238]:
```suggestion
    ```
```
https://gist.github.com/oke-py/d36b3b6014f3150b44988270d796204b#file-hello-minikube-diff-L222

File: content/ja/docs/tutorials/hello-minikube.md [hunk 14]
```diff
@@ -205,31 +208,35 @@ Minikubeはビルトインのアドオンがあり、有効化、無効化、あ
...
-    heapster was successfully disabled
+    etrics-server was successfully disabled
```
Comment (oke-py) [inline content/ja/docs/tutorials/hello-minikube.md:239]:
```suggestion
    metrics-server was successfully disabled
```

**Comment by ryokotmng** [comment]
Thanks for your comments! Will revise as you kindly suggested🙏 💦

File: content/ja/docs/tutorials/hello-minikube.md (hunk 3 above)
Comment (ryokotmng) [reply content/ja/docs/tutorials/hello-minikube.md:84]: Apologies🙇‍♂️

File: content/ja/docs/tutorials/hello-minikube.md (hunk 2 above)
Comment (ryokotmng) [reply content/ja/docs/tutorials/hello-minikube.md]: ありがとうございます🙇‍♂️

**Comment by ryokotmng** [comment]
@oke-py Revised and double-checked by myself🙏 you have very good eyes! Would be great if you could go through~:)

**Comment by inductor** [comment]
/approve
@ryokotmng Thank you for your contribution :)

**Pull request merged**